starburst-presto/presto.sh

```diff
@@ -123,23 +128,35 @@ function configure_hive(){
 connector.name=hive-hadoop2
 hive.metastore.uri=${metastore_uri}
 EOF
+
+# Add new connectors configs here 
+# For example: Adding jmx properties like
+  cat > presto-server-${PRESTO_VERSION}/etc/catalog/tpch.properties <<- "EOF"
+connector.name=tpch
+tpch.splits-per-node=4
+EOF
+
 }
 
 function configure_jvm(){
   cat > presto-server/etc/jvm.config <<EOF
 -server
 -Xmx${PRESTO_JVM_MB}m
--Xmn512m
--XX:+UseConcMarkSweepGC
--XX:+ExplicitGCInvokesConcurrent
--XX:ReservedCodeCacheSize=150M
+-XX:+UseG1GC
+-XX:-UseBiasedLocking
+-XX:G1HeapRegionSize=32M
+-XX:+UseGCOverheadLimit
 -XX:+ExplicitGCInvokesConcurrent
+-XX:+ExitOnOutOfMemoryError
```

Comment:
@ChethanUK these are good changes, thanks for making them.
However, we need some more fine tuning here.
Please extract this to a separate PR and cc me. (Or I can take over the jvm.config changes if you prefer this way.)

Reply:
@ChethanUK as we spoke offline -- i extracted this to https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/546 and update to be in line with current recommended options. 
I also noticed you added `-Djdk.attach.allowAttachSelf=true`  which has not been recommended yet, so I created https://github.com/prestosql/presto/pull/937.

I suggest that you revert your changes here to avoid conflicts.